admin/model_event_type_for_nds_group.go

```diff
@@ -1,992 +0,0 @@
-// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
-
-package admin
```

Comment:
When inline models are removed. I have separate jira to take care of that problem.